charts/kspm-collector/templates/_helpers.tpl

```diff
@@ -178,3 +178,16 @@ KSPM Collector nodeSelector
 true
 {{- end }}
 {{- end }}
+
+{{/*
+Return agent tags if available
+*/}}
+{{- define "agent.tags" }}
+{{- if hasKey .Values.global "settings" }}
+{{- if hasKey .Values.global.settings "tags" }}
+{{- .Values.global.settings.tags }}
+{{- end }}
+{{- else if hasKey .Values.agent.sysdig.settings "tags" }}
```

Comment:
I'm not 100% this will work. If the `kspm-collector` is installed as part of the `sysdig-deploy` AFAIK it will be able to read only `.Values.global` and `.Values.kspmCollector`. Can u please test it?

Reply:
@AlbertoBarba thanks. I'm actually relying on information I got from Lugo but I'm going to change this PR into a draft and ask for more guidance - There's a bit of a mess with how agent tags are supposed to be defined.